script.js

```diff
@@ -4,57 +4,65 @@ const bookInputOne = document.querySelector('#txtOne');
 const addBtn = document.querySelector('.add');
 const bookList = document.querySelector('.book-list');
 let bookArray = JSON.parse(localStorage.getItem('bookArray'));
```

Comment:
Array for books can only be used inside the `Books` class. Please define an array of books inside the class.